rest-service/manager_rest/rest/resources_v3_1/manager_config.py

```diff
@@ -49,3 +49,25 @@ def _authorization_config():
             'permissions': cfy_config.authorization_permissions
         }
         return authorization
+
+
+class ManagerRestConfig(SecuredResource):
+    @rest_decorators.exceptions_handled
+    @authorize('manager_rest_config_get')
+    def get(self):
+        """
+        Get the Manager's REST configuration
+        :return: Manager's REST configuration data
+        """
+        request_args = request.args.to_dict(flat=False)
+        current_app.logger.info('Retrieving Manager REST config. filter: {0}'.
+                                format(request_args))
+        result = dict()
+        filter_by = request_args.get('filter', [])
+        for key in filter_by:
+            result[key] = getattr(config.instance, key)
+        if not filter_by:
+            current_app.logger.debug('Retrieving Manager REST config without a'
+                                     ' filter')
+            result = config.instance.__dict__
```

Comment:
This is a dangerous practice. Using `__dict__` is usually discouraged, _especially_ if you're returning this object directly to the user.
e.g. https://stackoverflow.com/a/4727598/978089

Reply:
I agree, even though only a sys_admin can run this command.
So I think I'll create a subclass of Config in config.py that will override any unsafe information, that way it would seem like the key doesn't exist and we can still use the getattr function.

Reply:
I think it's more sensible to just have a `to_dict` method on the existing `Config` class.